brave-tests/src/main/java/brave/test/IntegrationTestSpanHandler.java

```diff
@@ -146,7 +146,9 @@ public MutableSpan takeLocalSpan() {
   MutableSpan doTakeSpan(boolean flushed) {
     MutableSpan result;
     try {
-      result = spans.poll(3, TimeUnit.SECONDS);
+      // Same rationale as ITRemote: tests should never take 5s in normal circumstances.
+      // In practice, @SpringBootTest tests overrun 3-5 seconds in CI.
+      result = spans.poll(20, TimeUnit.SECONDS);
```

Comment:
I always wanted this to be less, but I feel like moving it to 5s will just result in another PR to move it to 10

Reply:
So how about set to 10? ;)